packages/manager/modules/pci/src/projects/project/data-processing/jobs/submit-job/region-selector/translations/Messages_fr_CA.json

```diff
@@ -0,0 +1,4 @@
+{
+  "data_processing_regions_all": "Toutes les régions",
+  "data_processing_regions_select": "Sélectionnez une région"
```

Comment:
Dont add any translation to languages other than FR.  It will be taken care by translation process. So please remove the changes here.

Reply:
I'm not sure why github see this file as being new while seeing the other trad files as just moved but I didn't touched it, the component was just moved to another folder.